python/py-prompt_toolkit/Portfile

```diff
@@ -15,19 +12,19 @@ long_description    ${description}
 
 python.versions     27 34 35
 
-homepage            https://pypi.python.org/pypi/prompt_toolkit/
-master_sites        pypi:p/prompt_toolkit/
-distname            prompt_toolkit-${version}
+homepage            https://pypi.python.org/pypi/${python.rootname}/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
 
-checksums           rmd160  f7cfd594df441503de66cba763a5d7389b14a7d8 \
-                    sha256  805e026f0cbad27467e93f9dd3e3777718d401a62788c1e84ca038e967ad8ba2
+checksums           rmd160  0a4c43580e8227d2e7d1b3db9a1486f7e402c55b \
+                    sha256  b686ff216fc016dcbdf9ddf18d0ded480457213886ed4cda9fbc21002d18be54
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
-    depends_lib-append      port:py${python.version}-wcwidth \
-                            port:py${python.version}-pygments \
-                            port:py${python.version}-six
-    livecheck.type          none
+    depends_build-append  port:py${python.version}-setuptools
```

Comment:
This whitespace change violates the statement made in the modeline, namely that indentations shall be at multiples of 4 spaces.